perf/tests/propTypes.js

```diff
@@ -62,22 +68,24 @@ exports.fn = function(){
       bool: false,
       number: Math.random(),
       string: 'banana banana banana',
-      func: function() { return 'this is a function'; },
+      func: func,
       node: 'renderable string',
       node2: [MyReactComponent(), 'a string'],
-      instanceOf: new Thing,
+      instanceOf: new Thing(),
       element: MyReactComponent()
     }));
-  };
+  }
 
   React.render(List({
     array: [11, 12, 13, 14, 15, 16, 17, 18, 19, 1, 2, 3, 4, 5, 6, 7, 8, 9, 10],
     bool: false,
     number: Math.random(),
     string: 'banana banana banana',
-    func: function() { return 'this is a function'; },
+    func: function() {
+      return 'this is a function';
+    },
```

Comment:
Sure, but you created a `func` function and used that earlier to replace the exact same code block. Is there a reason you didn't make the above one an inline function declaration as well?